domain/src/main/java/org/oppia/android/domain/topic/TopicListController.kt

```diff
@@ -230,156 +211,20 @@ class TopicListController @Inject constructor(
-          lastCompletedChapterProgress.lastPlayedTimestamp
-        ) / ONE_DAY_IN_MS
-      val promotedStory = createPromotedStory(
-        storyId,
-        topic,
-        completedChapterProgressList.size,
-        story.chapterCount,
-        nextChapterSummary.name,
-        nextChapterSummary.explorationId
-      )
-      if (numberOfDaysPassed < ONE_WEEK_IN_DAYS) {
-        ongoingStoryListBuilder.addRecentStory(promotedStory)
-      } else {
-        ongoingStoryListBuilder.addOlderStory(promotedStory)
-      }
-    }
-  }
-
-  private fun createOngoingStoryListBasedOnRecentlyPlayed(
-    storyId: String,
-    story: StorySummary,
-    recentlyPlayerChapterProgress: ChapterProgress,
-    completedChapterProgressList: List<ChapterProgress>,
-    topic: Topic,
-    ongoingStoryListBuilder: OngoingStoryList.Builder
-  ) {
-    val recentlyPlayerChapterSummary: ChapterSummary? =
-      story.chapterList.find { chapterSummary ->
-        recentlyPlayerChapterProgress.explorationId == chapterSummary.explorationId
-      }
-    if (recentlyPlayerChapterSummary != null) {
-      val numberOfDaysPassed = (
-        oppiaClock.getCurrentCalendar().timeInMillis -
-          recentlyPlayerChapterProgress.lastPlayedTimestamp
-        ) / ONE_DAY_IN_MS
-      val promotedStory = createPromotedStory(
-        storyId,
-        topic,
-        completedChapterProgressList.size,
-        story.chapterCount,
-        recentlyPlayerChapterSummary.name,
-        recentlyPlayerChapterSummary.explorationId
-      )
-      if (numberOfDaysPassed < ONE_WEEK_IN_DAYS) {
-        ongoingStoryListBuilder.addRecentStory(promotedStory)
-      } else {
-        ongoingStoryListBuilder.addOlderStory(promotedStory)
-      }
-    }
-  }
-
-  private fun getStartedChapterProgressList(storyProgress: StoryProgress): List<ChapterProgress> {
-    return storyProgress.chapterProgressMap.values
-      .filter { chapterProgress ->
-        chapterProgress.chapterPlayState ==
-          ChapterPlayState.STARTED_NOT_COMPLETED
-      }
-      .sortedByDescending { chapterProgress -> chapterProgress.lastPlayedTimestamp }
-  }
-
-  private fun getCompletedChapterProgressList(storyProgress: StoryProgress): List<ChapterProgress> {
-    return storyProgress.chapterProgressMap.values
-      .filter { chapterProgress ->
-        chapterProgress.chapterPlayState ==
-          ChapterPlayState.COMPLETED
-      }
-      .sortedByDescending { chapterProgress -> chapterProgress.lastPlayedTimestamp }
-  }
-
   private fun createRecommendedActivityList(
     topicProgressList: List<TopicProgress>
   ): RecommendedActivityList {
     val recommendedActivityListBuilder = RecommendedActivityList.newBuilder()
 
     if (topicProgressList.isNotEmpty()) {
       val recommendedStoryBuilder = RecommendedStoryList.newBuilder()
+      // If initially only one topic is in progress populate combination of Ongoing story
+      // and Suggested stories.
```

Comment:
Here & below: we shouldn't be capitalizing ongoing/suggested/recent/other story list words unless they begin sentences or refer to actual classes or protos. Please update to ensure consistency.